client/js/models/archive.js

```diff
@@ -159,6 +172,7 @@ module.exports = {
             // XXX: Hack to fetch a small bit of data so size properly updates
           })
         }
+        send('archive:getMetadata', noop)
```

Comment:
am pretty sure this is the wrong signature for `send`, i think you might mean:
`send('archive:getMetadata', {}, noop)`